### PR TITLE
Fix middleware thunk test

### PR DESCRIPTION
**Pull request by danielrearden**

### CODE REVIEW

File: src/__tests__/http-test.js [hunk 1]
```diff
@@ -1117,12 +1117,8 @@ function runTests(server: Server) {
...
       urlString(),
       graphqlHTTP((req, res) => {
-        if (req) {
-          hasRequest = true;
-        }
-        if (res) {
-          hasResponse = true;
-        }
+        hasRequest = Boolean(req);
+        hasResponse = Boolean(res);
```
Comment (IvanGoncharov) [inline src/__tests__/http-test.js:1121]: Don't really like such conversions.
You can do:
```js
let seenRequest;

// ...
seenRequest = req;

// ...
expect(seenRequest) // ...
// Not sure how to use chai to check if it's requested, AFAIK it should be an instance of some class.
```

**Comment by IvanGoncharov** [comment]
@danielrearden You doing this to increase coverage?

**Comment by danielrearden** [comment]
@IvanGoncharov This fixes the issue we discussed [here](https://github.com/graphql/express-graphql/pull/640#discussion_r442084669). ESLint will complain about an unnecessary condition because the type indicates that `req` and `res` should always be defined.

By the way, as far as I can tell, istanbul is wrong -- the "else" branch is never actually hit.

**Comment by IvanGoncharov** [comment]
@danielrearden Thanks for the detailed explanation 👍

**Pull request merged**